### ISSUE TITLE
landmarks in 'my interactive sessions' page are off

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#673]
Landmarks in the my interactive sessions' page are inconsistent across browsers and somethings are landmarks that shouldn't be.

*  Compression options in iHPC app cards are landmarks for whatever reason. This is true in all browsers tested.
* The footer landmark is read as "content info landmark powered by Open Ondemand". It seems it should indicate it's the footer landmark.  In Edge it's only read as 'clickable'
* Edge differences
  * "connect to" buttons are landmarks. At least for `basic` apps (I guess the compression landmark overrides the button for the cards?)
  * navbar landmark is only read as "clickable". Other browsers read "Banner landmark Navigation landmark OSC ondemand link".
  * main landmark only reads "main landmark" others read "main landmark list with two items Home visited link"

Tested on NVDA + Windows + [Chrome, FF, Edge].

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737536074958) by [Unito](https://www.unito.io)

**Comment by ahmed-mgd** [comment]
Unable to replicate the following:
- Compression options in iHPC app cards are landmarks for whatever reason
- All Edge differences

**Comment by johrstrom** [comment]
This was fixed in #4254
